### ISSUE TITLE
NVDA 10-K "Item 1" does not provide the full item text

### ISSUE THREAD

**Issue by gchinna** [dgunning/edgartools#23]
NVDA 10-K **"Item 1"** does not seem to provide the full item text.
[February 24, 2023 - 10-K: Annual report for year ending January 29, 2023](https://www.sec.gov/ix?doc=/Archives/edgar/data/0001045810/000104581023000017/nvda-20230129.htm)

NVDA 10-K filing "Item 1" text spans from page 4 to page 15. However below code provides only part of "Item 1" text page 4 through middle of page 11.

```
tenk = Company("NVDA").get_filings(form="10-K").latest(1).obj()
print(f"NVDA item 1 text:\n{tenk['Item 1']}")

```
Is there another way to get the full **"Item 1"** text correctly?

**Comment by dgunning** [comment]
Hi,

I have found an issue where the library used for parsing the html - unstructured - does not handle inline xbrl tags. I am working to fix this and will add this 10-K to the test cases. Will update you on this.

Thanks

**Comment by dgunning** [comment]
Sorry for the long wait, the fix required a lot of rework. This is fixed in 2.10.1
